### PR TITLE
2018 Updates

### PR DESCRIPTION
**Pull request by ewdurbin**
Mostly a port of @sagesharp's work to Markdown

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -1,57 +1,78 @@
+# Code of Conduct
+
 PyCon US is a community conference intended for networking and collaboration in the developer community.
 
-We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by PyCon or not.
+We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by the Python Software Foundation or not.
 
 To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and volunteers at any PyCon event are required to conform to the following Code of Conduct. Organizers will enforce this code throughout the event.
```
Comment (ewdurbin) [inline code_of_conduct.md]: It's unclear to me what `delegates` mean in this context.

File: code_of_conduct.md [hunk 2]
```diff
@@ -1,57 +1,78 @@
...
 
-The Short Version
------------------
+## Our Standards
 
-PyCon is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of conference participants in any form.
+PyCon is dedicated to providing a harassment-free conference experience for everyone, regardless of age, gender identity and expression, sexual orientation, disability, physical appearance, body size, ethnicity, nationality, race, or religion (or lack thereof), education, or socio-economic status.
 
-All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any conference venue, including talks.
+Examples of behavior that contributes to creating a positive environment include:
```
Comment (ewdurbin) [inline code_of_conduct.md]: @sagesharp Do we have any good source material for adding a few more points to this list? It feels a tad imbalanced against the following list.

**Review by ejodlowska — COMMENTED** [review]
<tbd> email can be changed to pycon-us-report@python.org

File: code_of_conduct.md (hunk 1 above)
Comment (sagesharp) [reply code_of_conduct.md]: We can remove the word if it doesn't make sense. Perhaps "staff" instead of delegates?

File: code_of_conduct.md (hunk 2 above)
Comment (sagesharp) [reply code_of_conduct.md]: Sure! I only left in the positive examples that were already in the document.

The [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) has some short social guidelines:
 * Using welcoming and inclusive language
 * Being respectful of differing viewpoints and experiences
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members

The [Django Project](https://www.djangoproject.com/conduct/) has some more detailed social guidelines, but most of them are pretty well summarized by the above points.

The [Recurse Center](https://www.recurse.com/code-of-conduct) has a set of social rules that are geared towards being very newcomer-friendly:
 * **No feigning surprise.** The first rule means you shouldn't act surprised when people say they don't know something. This applies to both technical things ("What?! I can't believe you don't know what the stack is!") and non-technical things ("You don't know who RMS is?!"). Feigning surprise has absolutely no social or educational benefit: When people feign surprise, it's usually to make them feel better about themselves and others feel worse. And even when that's not the intention, it's almost always the effect. As you've probably already guessed, this rule is tightly coupled to our belief in the importance of people feeling comfortable saying "I don't know" and "I don't understand."
 * **No well-actually's.** A well-actually happens when someone says something that's almost - but not entirely - correct, and you say, "well, actually…" and then give a minor correction. This is especially annoying when the correction has no bearing on the actual conversation. This doesn't mean we aren't interested in truth-seeking or that we don't care about being precise. Almost all well-actually's in our experience are about grandstanding, not truth-seeking. (Thanks to Miguel de Icaza for originally coining the term "well-actually.")

Depending on how newcomer-friendly you want the PyCon sprints to be, you might consider adopting the two rules from the Recurse Center. I would encourage adopting the bullet points from the Contributor Covenant at the very least.

File: Staff Procedure for incident handling.md [hunk 3]
```diff
@@ -1,65 +1,122 @@
...
+* Requiring that a person  immediately leave the event and not return
+* Banning a person from future events (either indefinitely or for a certain time period)
+* Removing a person  from membership of relevant organizations
+* Publishing an account of the incident and calling for the resignation of a person from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar, typically event staff have sufficient governing rights over their space that this isn't as useful)
 
- - What happened?
- - Are we doing anything about it?
- - Who is doing those things?
- - When are they doing them?
+Give reported person a place to appeal to if there is one. Keep in mind that it is not a good idea to encourage an apology from the harasser.
```
Comment (sagesharp) [inline Staff Procedure for incident handling.md]: "Give reported person a place to appeal to if there is one"
should be
"If a reported person wants to appeal the decision, they may contact the Python Software Foundation Board <psf@python.org>."

"Keep in mind that it is not a good idea to encourage an apology from the **harasser**."
should be
"Keep in mind that it is not a good idea to encourage an apology from the **reported person**."

File: Attendee Procedure for incident handling.md [hunk 4]
```diff
@@ -1,38 +1,64 @@
...
+**All reports will be kept confidential**. When we discuss incidents with people who are reported, we will anonymize details as much as we can to protect reporter privacy.
+
+However, some incidents happen in one-on-one interactions, and even if the details are anonymized, the reported person may be able to guess who made the report. If you have concerns about retaliation or your personal safety, please note those in your report. We still encourage you to report, so that we can support you while keeping our conference attendees safe. In some cases, we can compile several anonymized reports into a pattern of behavior, and take action on that pattern.
+
+In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+## Report Handling Procedure
+
+When you make a report to an incident responder, they will gather information about the incident according to the PyCon US [Staff Procedure For Incident Response] (https://us.pycon.org/2018/about/code-of-conduct/staff-procedure/).
```
Comment (sagesharp) [inline Attendee Procedure for incident handling.md]: Space in between the link syntax here.

File: code_of_conduct.md (hunk 2 above)
Comment (ewdurbin) [reply code_of_conduct.md]: Thank you @sagesharp! The Recurse Center social rules are of my new favorites (I recently attended a batch). I think they are _strongly_ effective, but am concerned that they do take a little bit more introduction than we can reasonably allot.

I'll add the Contributor Covenant points, and we could consider the Recurse Center's social rules for inclusion in time.

**Comment by ewdurbin** [comment]
@ejodlowska @sagesharp this is ready for a final round of review.

**Comment by ewdurbin** [comment]
@ejodlowska can you provide the final room names/numbers and purpose that we have for PyCon 2018? I think making sure those are up to date in the document and I think that's the last remaining open question.

**Comment by ejodlowska** [comment]
Sorry for the delay. Room 23 on May 11 and Room 25C May 12-13.

**Review by Mohammadseyfi1984 — APPROVED** [review]

**Pull request merged**